release/README.md

```diff
@@ -86,25 +91,29 @@ discarded as they are uninteresting to our users.
 
 ### Send a PR with all these changes
 
-It is not recommended that you create the release branch before this PR is
-*merged*, but in some circumstances it might be needed, for example, if a large
-change that could destabilize the release is about to be merged, or if we want
-to create the release at an specific point in the revision history.
+In general, do not create the release branch before this PR is *merged*. We want
+to create the release from a stable point in the default branch (`main`), and
+we want this point to include the updated release notes and API baselines.
+There may be exceptions to this guideline, you are encouraged to use your own
+judgement.
```

Comment:
judgment

Reply:
Fixed, let's hope we have no customers used to British spelling.